bigquery.js

```diff
@@ -0,0 +1,93 @@
+var BigQueryClient = require('google-cloud/node_modules/@google-cloud/bigquery');
+
+const projectData = require('./auth.json');
+
+const config = {
+  projectId: projectData['project_id'],
+  keyFilename: './auth.json'
+};
+
+const ds = 'atomic';
+
+const PARTITION_FIELDS = ['updated_at', 'created_at', 'fired_at', 'ts'];
+
+var BigQuery = function() {
+  this.conn = BigQueryClient(config);
+};
+
+BigQuery.prototype.insertInto = function(tableName, rows, options) {
+  //Here I should get the table based on name
+  // and insert in the right partition whenever the table is partitioned
+  rows.forEach((row) => {
+    // First, we get the table object with all available information
+    this.table(tableName, function(err, table) {
+      if (err) {
+        console.log(`Sorry! We've got an error while fetching the table ${tableName}.\nERROR: ${err}`);
+        return;
+      }
+      var isPartitioned = BigQuery.isPartitioned(table);
+      var tablePartition = isPartitioned ? BigQuery.partition(row, table) : undefined;
```

Comment:
@diegospereira the method `BigQuery.partition` returns the partition name? If yes, you can give a better name for this method. Some suggestions: `BigQuery.partitionName` ? or `BigQuery.partitionDecorator` ?

Using only `partiton`, for me sounds like this method will return some object, not a string with partition name

Reply:
Yes, it returns partition name whenever the table is partitioned, otherwise it returns the tableName itself. https://github.com/Bidu/snowplow-bigquery-lambda-loader/pull/3/commits/0e63308ff21fbbf64c5a528318f9ae468c02ea21